blake2b224/blake2b224.go

```diff
@@ -0,0 +1,24 @@
+package blake2b224
+
+import (
+	"fmt"
+	"golang.org/x/crypto/blake2b"
+)
+
+// Size224 is the length of blake2b224 hashes in bytes.
+const Size224 = 224 / 8
+
+// Sum224 returns the blake2b224 hash of the data.
+func Sum224(data []byte) ([Size224]byte, error) {
+	blake2b224, err := blake2b.New(Size224, nil)
+	if err != nil {
+		return [Size224]byte{}, fmt.Errorf("unable to instantiate blake2b224: %w", err)
+	}
+	_, err = blake2b224.Write(data)
+	if err != nil {
+		return [Size224]byte{}, fmt.Errorf("unable to compute blake2b224 hash of data: %w", err)
+	}
+	var res [Size224]byte
+	copy(res[:], blake2b224.Sum(nil))
```

Comment:
Since the function already does error checking is responsible for a valid hash generation, I would check the returnvalue for `copy` to match `Size224`.